Fix PHP error the 'daemon' tab of Icinga Web -> Icinga Director on systems
without /proc.

Index: library/Director/Web/Widget/BackgroundDaemonDetails.php
--- library/Director/Web/Widget/BackgroundDaemonDetails.php.orig
+++ library/Director/Web/Widget/BackgroundDaemonDetails.php
@@ -114,7 +114,7 @@ class BackgroundDaemonDetails extends BaseHtmlElement
                         $pid
                     ],
                     Html::tag('pre', $process->command),
-                    Format::bytes($process->memory->rss)
+                    $process->memory === false ? 'n/a' : Format::bytes($process->memory->rss)
                 ]));
             }
             $this->add($table);
